media/starboard/sbplayer_bridge.cc

```diff
@@ -1143,9 +1143,7 @@ void SbPlayerBridge::OnPlayerStatus(SbPlayer player,
     if (sb_player_state_initialized_time_.is_null()) {
       sb_player_state_initialized_time_ = Time::Now();
     }
-    sbplayer_interface_->Seek(player_, preroll_timestamp_, ticket_);
-    SetVolume(volume_);
-    sbplayer_interface_->SetPlaybackRate(player_, playback_rate_);
+    PrepareForSeek();
```

Comment:
`sbplayer_interface_->Seek(player_, preroll_timestamp_, ticket_);` might be needed for suspend/resume support, where we have to seek to the previous playback position on resume.

Consider adding a TODO here, referring to the suspend/resume bug.

Reply:
Done